src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -91,7 +97,7 @@ private Object doMakeObject(String name)
                 return createObject(className);
             }
             catch (final CheckstyleException ex) {
-                // keep looking
+                LOG.debug("Keep looking, ignoring exception", ex);
```

Comment:
no need merge anything, that block are not related even they looks close on diff